internal/format/markdown_table.go

```diff
@@ -17,6 +17,19 @@ const (
 		{{- end -}}
 	{{ end -}}
 	`
+	tableResourcesTpl = `
+	{{- if .Settings.ShowResources -}}
+		{{ indent 0 "#" }} Resources
+		{{ if not .Module.Resources }}
+			No resources.
+		{{ else }}
+			The following resources are used by this module:
+			{{- range .Module.Resources }}
+				- [{{ .FullType }}]({{ .URL }})
```

Comment:
I think it might be nicer (and more consistent with the rest of the sections) that this will be structured as a Markdown table format rather than a bullet list. Maybe just a single column (e.g. `Name`)?

Reply:
I'm good with that.